### DISCUSSION TITLE
Copilot for VS (not code) duplicates the keys entered

### DISCUSSION THREAD

**Discussion by stevenxi** [community/community#14538] [Copilot Conversations]
With Copilot for VS extension, sometime, when press one key, it automatcly inserts 4 keys.
For the .cs file, only happened (sometime) to the tab key.
But for the .cshtml, it could happens to any key.

https://user-images.githubusercontent.com/3340119/162398103-4f3c6323-9804-40ec-8342-74b70952560e.mp4

**Comment by andybeaulieu** [comment]
this other post has a temporary workaround - https://github.com/github-community/community/discussions/14206

if you click the github settings you can disable html projections --- but note that, in my case I had to click the settings icon a couple of times before I saw the "Enable html projections" option

![image](https://user-images.githubusercontent.com/6075659/173065530-b137f37b-13b1-4461-ac7b-a9021a2c3704.png)
